Pod/Classes/WavesLayer.swift

```diff
@@ -114,56 +84,30 @@ class WavesLayer: CALayer {
         }
         
         // Draw new layers
-        for num in (0..<_numberOfWaves).reverse() {
+        for num in (0..<_numberOfWaves).reversed() {
             let calculatedRadius = CGFloat(num + 1) * _distanceBetweenWaves
-            let radius = calculatedRadius + (calculatedRadius * CGFloat(_sonarView.sonarViewLayout.waveRadiusOffset(_sonarView)))
+            let radius = calculatedRadius + (calculatedRadius * CGFloat(_sonarView.sonarViewLayout.waveRadiusOffset(sonarView: _sonarView)))
             let layer = self.circleWithRadius(radius: radius)
-            
-            let strokeAnimation = CABasicAnimation(keyPath: "strokeEnd")
-            strokeAnimation.duration = 0.3
-            strokeAnimation.beginTime = CACurrentMediaTime() + Double(num) * 0.3
-            strokeAnimation.fromValue = 0.0
-            strokeAnimation.toValue = 1.0
-            strokeAnimation.removedOnCompletion = false
-            strokeAnimation.fillMode = kCAFillModeForwards
-            
-            layer.addAnimation(strokeAnimation, forKey: "strokeEnd")
-            
-            layer.strokeEnd = 0.0
-            
             layer.frame = self.bounds
             self.addSublayer(layer)
             _waveLayers.append(layer)
             
-            let gradientColors = [UIColor.whiteColor().CGColor, UIColor.whiteColor().CGColor, SonarView.lineShadowColor.CGColor]
+            let gradientColors = [UIColor.white.cgColor, UIColor.white.cgColor, SonarView.lineShadowColor.cgColor]
             let gradientSize = 1.0 - (18 / radius)
             let gradientLocations: [CGFloat] = [0.0, gradientSize, 1.0]
-            let gradient = RadialGradientLayer(frame: self.frame, radius: radius - 0.5, center: CGPointMake(CGRectGetWidth(self.frame) / 2, CGRectGetHeight(self.frame)), colors: gradientColors, locations: gradientLocations)
-            
-            let displayAnimation = CABasicAnimation(keyPath: "opacity")
-            displayAnimation.duration = 0.3
-            displayAnimation.beginTime = CACurrentMediaTime() + Double(num) * 0.3
-            displayAnimation.fromValue = 0.0
-            displayAnimation.toValue = 1.0
-            displayAnimation.removedOnCompletion = false
-            displayAnimation.fillMode = kCAFillModeForwards
-            
-            gradient.addAnimation(displayAnimation, forKey: "opacity")
-            
-            gradient.opacity = 0.0
```

Comment:
Oh, this was taken from 2.0.1 not from master... it needs to be resolved with master